seqr/views/utils/terra_api_utils.py

```diff
@@ -110,12 +113,23 @@ def list_anvil_workspaces(user, fields=None):
 
 def user_get_workspace_access_level(user, workspace_namespace, workspace_name):
     path = "api/workspaces/{0}/{1}?fields=accessLevel".format(workspace_namespace, workspace_name)
+
+    cache_key = 'terra_req__' + hashlib.md5("{}__{}".format(user, path).encode('utf-8')).hexdigest()
```

Comment:
What value is there to hashing this?

Reply:
It used an access token as part of the key. The hash can protect the token from being logged. Now, it doesn't include sensitive information in the key. So I can get rid of the hash.